kraken-build/src/kraken/build/experimental/python/__init__.py

```diff
@@ -0,0 +1,86 @@
+from collections.abc import Sequence
+from typing import Any
+from kraken.build.experimental.python.targets import PythonApp, PythonPex, PythonRequirements, PythonSources
+
+from kraken.core.system.target import NamedTarget
+
+
+def python_sources(
+    name: str,
+    sources: list[str],
```

Comment:
Why not
```suggestion
    sources: Sequence[str],
```
here too?